drivers/disk/nvme/nvme_helpers.h

```diff
@@ -0,0 +1,390 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) 2022 Intel Corp.
+ */
+
+#ifndef ZEPHYR_DRIVERS_DISK_NVME_NHME_HELPERS_H_
+#define ZEPHYR_DRIVERS_DISK_NVME_NHME_HELPERS_H_
+
+#define NVME_GONE		0xfffffffful
```

Comment:
Gone? Maybe this is a NVMe defined parameter but from the name to me as someone that doesn't know the NVMe spec at all, I'm not really sure what this value is for